testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/partialexport/PartialExportTest.java

```diff
@@ -15,11 +16,8 @@
 import org.keycloak.testsuite.ProfileAssume;
 import org.keycloak.testsuite.admin.AbstractAdminTest;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can you please remove the star import? We don't use them in Keycloak.

Reply:
sure, I guess my IDE did that when I used Optional

Reply:
Can you also squash the commits please? Then it's ready from my POV.

Reply:
Done!